src/chat.rs

```diff
@@ -2211,7 +2211,8 @@ async fn prepare_msg_blob(context: &Context, msg: &mut Message) -> Result<()> {
         if msg.viewtype == Viewtype::Image || maybe_sticker {
             blob.recode_to_image_size(context, &mut maybe_sticker)
                 .await?;
-            if !maybe_sticker {
+
+            if !maybe_sticker && !msg.param.exists(Param::ForceSticker) {
```

Comment:
It should be checked in `recode_to_size()` to skip all the image-related logic

Reply:
You mean we can just savely short circuit here: https://github.com/deltachat/deltachat-core-rust/blob/37383c10ac45c3119d83631761d4a13e9e11bf70/src/blob.rs#L410 because no more processing is needed?

Reply:
Yes, we don't need to recode stickers, at least it's the current approach. We can short-circuit it here, yes, but it would be some duplication of the existing logic, so i'd prefer to make `recode_to_size()` do all necessary checks